packages/dd-trace/src/appsec/iast/analyzers/sql-injection-analyzer.js

```diff
@@ -6,7 +6,7 @@ class SqlInjectionAnalyzer extends InjectionAnalyzer {
     super('SQL_INJECTION')
     this.addSub('apm:mysql:query:start', ({ sql }) => this.analyze(sql))
     this.addSub('apm:mysql2:query:start', ({ sql }) => this.analyze(sql))
-    this.addSub('apm:pg:query:start', ({ query }) => query && this.analyze(query.text))
+    this.addSub('apm:pg:query:start', ({ statement }) => this.analyze(statement))
```

Comment:
why is it a different name then above ?

Reply:
`query` is an object that can be manipulated by pg plugin. `statement` is a new property that maintains the customers original query.